ebu_tt_live/documents/ebuttd.py

```diff
@@ -46,8 +52,23 @@ def create_from_raw_binding(cls, binding):
         instance._ebuttd_content = binding
         return instance
 
+    def _get_bds(self):
```

Comment:
it is a private method inteed as it is _get_bds. Not meant for people to tamper with :D